lib/augmentConfig.js

```diff
@@ -3,7 +3,6 @@
 const configurationError = require('./utils/configurationError');
 const getModulePath = require('./utils/getModulePath');
 const globjoin = require('globjoin');
-const merge = require('deepmerge');
```

Comment:
[suggestion] Oh, it seems we now use `deepmerge` only for testing. Can we move the package to `devDependencies`?

```console
$ git grep deepmerge
lib/__tests__/disableRanges.test.js:4:const merge = require('deepmerge');
lib/testUtils/mergeTestDescriptions.js:3:const merge = require('deepmerge');
package-lock.json:16:        "deepmerge": "^4.2.2",
package-lock.json:2839:    "node_modules/deepmerge": {
package-lock.json:2841:      "resolved": "https://registry.npmjs.org/deepmerge/-/deepmerge-4.2.2.tgz",
package-lock.json:5385:        "deepmerge": "^4.2.2",
package-lock.json:14726:    "deepmerge": {
package-lock.json:14728:      "resolved": "https://registry.npmjs.org/deepmerge/-/deepmerge-4.2.2.tgz",
package-lock.json:16651:        "deepmerge": "^4.2.2",
package.json:114:    "deepmerge": "^4.2.2",
```

https://github.com/stylelint/stylelint/blob/5d7695af46f02bf380b47e358173aeb190c81e97/package.json#L114

Reply:
You always spot all these subtle details! Great idea! Fixed.

Reply:
Thank you. 9eef5ac looks good! 👍🏼